Frameworks/UIKit/UIColor.mm

```diff
@@ -504,7 +505,7 @@ - (UIColor*)initWithPatternImage:(UIImage*)image {
 callbacks.releaseInfo = 0;
 callbacks.drawPattern = __UIColorPatternFill;
 
-_pattern = (id) CGPatternCreateColorspace(self, bounds, m, bounds.size.width, bounds.size.height, 0, FALSE, &callbacks, pImg->_has32BitAlpha ? _ColorRGBA : _ColorRGB);
+_pattern = (id) CGPatternCreateColorspace(self, bounds, m, bounds.size.width, bounds.size.height, 0, FALSE, &callbacks, pImg->_has32BitAlpha ? _ColorABGR : _ColorBGR);
```

Comment:
This change got hit as part of the surface enum renaming. I did it elsewhere for similar code that was commented out or compiled out via ifdefs just for consistency.